content/en/graphing/metrics/explorer.md

```diff
@@ -11,21 +11,56 @@ further_reading:
     text: "Metrics Distributions"
 ---
 
-Use the [Metrics Explorer page][1] to begin examining the [metrics][2] collected from your application by Datadog.
+## Overview
 
-Clicking into the "Graph" text box brings up a list of all the metrics that Datadog gathers from your application. You can also search the list by typing into the box. Selecting metrics populates real-time graph visualizations on the right hand side. By default, up to 20 graphs are shown. You can change this number by clicking on the "Options" gear.
+The [Metrics Explorer][1] is a basic interface for examining your metrics in Datadog. For more advanced options, create a [notebook][2], [screenboard][3], or [timeboard][4].
 
-{{< img src="graphing/metrics/explorer/explorer.png" alt="Metrics Explorer" responsive="true" style="width:60%;" >}}
+## Graphing
 
-"Over" allows you to narrow your results by tag. Under "One graph per", you can choose to split a metric into multiple graphs by host, container, region, team, and more.
+Click on the **Graph** text box to see a list of metrics you have sent to Datadog. Search the list by typing in the text box. Select a metric to populate a real-time graph visualization on the right side. Each metric you select creates a separate graph. Above the graphs, you can specify the time frame and graph size.
 
-{{< img src="graphing/metrics/explorer/split-by-team.png" alt="Split By Team" responsive="true" style="width:60%;">}}
+{{< img src="graphing/metrics/explorer/graphs.png" alt="Metrics Explorer" responsive="true" style="width:80%;" >}}
 
-You can also export these graphs to new and existing timeboards.
+**Note**: The **Calculate as count where applicable** check box appears for metrics with the `RATE` type.
+
+### Scope
+
+Define scope with the **Over** text box by selecting or searching for tags. For example, see only the metrics for a specific host, cluster, environment, region, etc.
+
+### Grouping
+
+Define grouping with the **One graph per** text box by selecting or searching for tag keys. For example, split a metric into multiple graphs by host, container, region, environment, etc. Metrics tagged with the `<KEY>:<VALUE>` format can be grouped. 
+
+### Space aggregation
+
+Define [space aggregation][5] with the **On each graph, aggregate with the** text box. The options are:
```

Comment:
Define the [space aggregation][5] used to combine a metric's values with the **On each graph, aggregate with the** text box. The options are: